artichoke-backend/src/extn/core/kernel/kernel.rb

```diff
@@ -68,13 +68,14 @@ def Hash(arg) # rubocop:disable Naming/MethodName
       return ret if ret.is_a?(Hash)
 
       raise TypeError, "can't convert #{classname} into Hash" if ret.nil?
+
       raise TypeError, "can't convert #{classname} to Hash (#{classname}#to_hash gives #{ret.class})"
     end
 
     raise TypeError, "can't convert #{classname} into Hash"
   end
 
-  def Integer(arg, base = 0, exception: true) # rubocop:disable Naming/MethodName
```

Comment:
this is ok since we're planning on ripping this impl out and pulling it into Rust.